### ISSUE TITLE
jsperf.com returning 502s

### ISSUE THREAD

**Issue by JaKXz** [jsperf/jsperf.com#537]
Hi, 

I found #418 but didn't want to comment on a very old issue, but I am seeing 502s for https://jsperf.com 

If it helps, here is the cloudflare error screen that I see in every major browser:

![image](https://user-images.githubusercontent.com/2642545/93804176-ebde1600-fc13-11ea-8b0f-3d80491d02e0.png)

**Comment by thomasjammet** [comment]
+1

**Comment by MathieuPOUX** [comment]
Same issue here too

**Comment by Mr-jiangzhiguo** [comment]
+1

**Comment by danawoodman** [comment]
Lol this is still an issue. So jsPerf is a dead service I assume?

**Comment by asilluron** [comment]
Hi Dana, we're working to resolve. Apologies for delay

**Comment by nikitahl** [comment]
Are there any updates on this issue? It's the 4th of November and it's still down (502 error).

**Comment by DoctorDerek** [comment]
Hey @asilluron Thanks for such a great service! I love JSPerf 🥇

**Comment by danawoodman** [comment]
@asilluron site is still down a month later 😢

**Comment by wortwart** [comment]
+1. I haven't been able to access jsperf.com for several weeks - with different devices, with or without VPN. And that is really a problem as I'm doing research about web performance. I wish there was something like a GitHub repo with your user-generated test cases so that I could at least run them on my own.

**Comment by jwdunn1** [comment]
@wortwart Consider using https://jsbench.me in the interim.

**Comment by EricRabil** [comment]
If jsperf is not coming back, could we please have a database dump? A lot of helpful benchmarks in there that will take time to replenish elsewhere.

**Comment by peder1001** [comment]
Yes, I too loved jsperf for the benchmarks. Would love to see this running again.

**Comment by FredZeng** [comment]
+1, waiting for any good news.

**Comment by TwitchBronBron** [comment]
I'd even settle for a version of jsperf that just runs in the browser with no backing database at the moment. It's not overly difficult to slap together some benchmarks for certain things. Could you stand up jsperf again with the server-side components (log-in, finding old saved items, etc...) disabled so we can at least run benchmarks? I strongly dislike the UI of all the other performance websites out there.

**Comment by VictorQueiroz** [comment]
I just wonder why exactly jsperf is dead. Why not hand the project to someone that's interested in taking care of it? It seems like a useful and important project for the community.

**Comment by DoctorDerek** [comment]
That's the $10,000 dollar question, Vic.

Haha 😂

I've tried https://jsbench.me and https://jsben.ch and prefer the latter,
but JSPerf has a sweet spot in my heart.

**Comment by TwitchBronBron** [comment]
I decided to write a serverless client-side-only clone of jsPerf. It's a copy of the edit and test pages from jsperf. I still have a few small tweaks to make, but I left the existing jsPerf benchmarking logic intact. 

jsperf-static stores your test results in local storage, so there's currently no sharing performance result URLs. However, I plan on writing "generate share url" functionality that will compress your test data so it can fit on a URL. Obviously larget tests won't fit, but for the small ones it should work just fine. 

Here's the site: https://twitchbronbron.github.io/jsperf-static/

And here's the repository, feel free to open issues for anything you find that is missing or broken: https://github.com/TwitchBronBron/jsperf-static

Give it a try, I'd love to know what you think!

**Comment by EricRabil** [comment]
It'd be so nice if we could have a database dump – I frequently come across posts referring to a JSPerf that I can no longer play around with. Really miss this tool.

**Comment by Mathias54** [comment]
hello @asilluron there is any update about the problem?

**Comment by asilluron** [comment]
Thank you for all the feedback. It seems that problem has to do with the
"routing" rules on the Vercel platform. It was not my intention to let this
sit so long, my sincere apologies, I've been helping with projects related
to front-line Covid problems (food delivery, pickup, mortgage servicing,
etc). I think we just need to tweak the Vercel config and see what happens.

**Comment by Mathias54** [comment]
Thank you for your answer. There is something I can help to make it works?

**Comment by the-wazz** [comment]
fyi, website still down.

**Comment by mathiasgheno** [comment]
The error for me is different now. 

![Screenshot from 2021-02-21 19-52-52](https://user-images.githubusercontent.com/14915619/108641546-8326ee00-747e-11eb-97ff-67aef5f76584.png)

**Comment by CubeyTheCube** [comment]
site is still down

**Comment by TwitchBronBron** [comment]
Yes, still down for me as well. 

@asilluron, we realize you're busy, but you're the only one who can fix this. Any chance you could carve out a little time in your schedule to give this another look?

**Comment by VictorQueiroz** [comment]
@asilluron (or whoever owns this project) just make the jsperf database publicly available and let us do what you can't. That's the best you can do for the community right now.

**Comment by asilluron** [comment]
@VictorQueiroz I don't own the project, I am just a collaborator, but I have set some time aside. I apologize, work has been beyond insane. I was in the midst of deploying online ordering for a major grocery chain during Covid, and then it just grew from there. 

I have opened a PR to track the work https://github.com/jsperf/jsperf.com/pull/541

**Comment by Lagicrus** [comment]
> @VictorQueiroz I don't own the project, I am just a collaborator, but I have set some time aside. I apologize, work has been beyond insane. I was in the midst of deploying online ordering for a major grocery chain during Covid, and then it just grew from there.
> 
> I have opened a PR to track the work #541

In the effort of logging and making sure everyone is in the now, after I saw this comment I went to test the site, just to check.
Found this error 
![image](https://user-images.githubusercontent.com/24227350/123757759-35037a80-d8b6-11eb-94cc-1c1cde7ceb68.png)

So it seems to have shifted somewhat in the error message, which is progress?

**Comment by TwitchBronBron** [comment]
Now there's a new error. **402: PAYMENT_REQUIRED**. Is there any chance of this project ever coming back online? If not, please just tell us so we can move on with our lives. 

![image](https://user-images.githubusercontent.com/2544493/132571924-8d7628be-f87d-4e35-a058-25d7aaff3725.png)

**Comment by VictorQueiroz** [comment]
To be honest, it seems so selfish to have everyone just wondering what is going with this project while whoever is in charge is just hiding somewhere. I believe we'd all appreciate getting some understanding on whether this project will ever be brought back on or not.

Are you guys expecting to get some money out of this? Please enlighten us with some sort of information about why this project was completely abandoned to a point where no one understands what is going on anymore.

I might be interested in buying this project, please let me know who can decide on this. Feel free to contact me anytime.

**Comment by EricRabil** [comment]
@maxbeatty @asilluron pretty please ❤️

**Comment by reverofevil** [comment]
@asilluron A friend reminder that epidemic is over for quite a while, but the website is still down. :)

Do you have an access to the database? Could you please upload it somewhere?

**Comment by TwitchBronBron** [comment]
It appears that someone has stood up a new site in the spirit of jsperf.com called https://jsperf.app . They've also retrieved most or all of the old tests from archive.org, so any dead jsperf.com links can be changed to jsperf.app and they'll seemingly work?

Here's an issue explaining how they handled the legacy tests:

https://github.com/rd13/jsperf.app/issues/39

At this point, I'm assuming jsperf will probably never come back. It would be nice to have the domain given to the jsperf.app project instead so all the old links can come alive again. Someone's still paying the domain registrar fees...
